etc/acme/templates/cert.ctmpl

```diff
@@ -1 +1,3 @@
-{{if key "nginx/acme/cert"}}{{key "nginx/acme/cert"}}{{end}}
+{{ $service_name := env "SERVICE_NAME" }}
+{{ $service_name := or $service_name "nginx" }}
```

Comment:
I'm not saying the following with certainty, but there's a chance we'll also (or alternatively) want to set `SERVICE_NAME` to `nginx` as a default value in the Dockerfile. That would result in reliably getting a env var, even if the user doesn't supply one.

If I remember correctly, that syntax would look like:

```
ENV SERVICE_NAME =${SERVICE_NAME:-nginx}
```